assets/javascripts/modules/ajaxCallbacks.js

```diff
@@ -121,16 +121,19 @@ var ajaxCallbacks = {
         var $action = $('<td></td>').addClass('text--right hard--right')
 
         var $form = $('<form></form>')
-          .attr('action', $list.data('collaborator-remove-url') + email)
+          .attr('action', $list.data('collaborator-remove-url'))
           .attr('data-ajax-submit', true)
           .attr('data-callback-args', '')
           .attr('data-callback-name', 'apiCollaboratorRemoveResponse.callbacks')
-          .addClass('form')
+          .attr('data-container', '.js-remove-collaborator-data')
```

Comment:
I've tried changing but it breaks the form submission.  Looking at the example in ajaxFormSubmit.js it shows #Selector (line 26). That makes me think a dot or hash is required